README.md

```diff
@@ -1,16 +1,251 @@
+      `csv_data`
+  - Instance methods
+    - `#common_number_of_trials`: Because the database crashed, we're not sure
+      how many trials are common to the participants. We need this class to
+      tell us the minimum _common_ number of trials.
+      ```csv
+        1,Arya,6,"1,3"
+        2,Arya,6,"2,3"
+        3,Arya,6,"3,3"
+        4,Arya,6,"4,3"
+        5,Byron,6,"1,3"
+        6,Byron,6,"2,3"
+        7,Byron,6,"3,3"
+        8,Byron,6,"4,3"
+        9,Byron,6,"5,3"
+        10,Carl,6,"1,3"
+        11,Carl,6,"2,3"
+        12,Carl,6,"3,3"
+      ```
+      In this CSV file, we have a `common_number_of_trials` of `3`. "Carl" has the
+      fewest trials and so, if we want to compare candidates equally as a
+      percentage, we need the denominator to be **`3`**
+    - `#least_trials_candidate`: Which candidate had the least, and thus
+      common, number of trials? Returns a `String`
+    - `#most_trials_candidate`: Which candidate had the most trials? Returns a
+      `String`
+
+> **TYPOGRAPHICAL NOTE**: Developer documentation uses `#methodname` to show that
+> `methodname` is an instance method. They use `::methodname` to denote a class
+> method. For trigonometry equations there's `Math::sin`, for instances of
+> `Mankind` there's `Mankind#sin`.
+
+Use the command `rspec spec/luck_analyzer_spec.rb` from the terminal and get
+all of these tests passing.
+
+### Second Release: Who is the Luckiest?
+
+The DoLS would like for you to determine the following:
+
+1.  "Who is the luckiest candidate?" Where:
+2.  "Lucky" is defined as having had a dice roll where the total pips is equal
+    to `7`
+3.  "Luckiest" is defined as having the most "lucky" rolls within a set of
+    size `common_number_of_trials`. We should take as many "lucky" rolls as
+    possible
+    - Example
+    - Gven a common size of 3 (where "L" means "Lucky" and "U" means "Unlucky"):
+    - A has L, L, L, L => [L, L, L] => 3/3 => 100%
+    - B has U, U, U, L => [L, U, U] => 1/3 => 33%
+    - C has U, U, U, U, U, U, U, U, U, L, U, L => [L, L, U] => 66%
+    - "A" is the luckiest!
+
+The (DoLS) famously asks for new features often, so we want to build this using
+an OO. DoLS have certified that the current code of `Die` and `DiceRoller` are
+very lucky. They have requested, and paid you handsomely, to update the
+existing classes (provided in `lib/`). The trick is that _new_ features can't
+break _existing_ functionality. They have given you an existing test suite.
+
+> You can run the full test suite (including your work to satisfy
+> `luck_analyzer_spec`, by running `rspec` from the Terminal or `learn`).
+
+- Update `Die` so that:
+
+  - It can be initialized with a permanent value. When a set value is present, all
+    calls to the `#roll` instance method return the set value. The permanent
+    value should be accessible via a call to `set_value`
+  - It provides a `#random_roll?` method which returns `true` if the `Die`'s
```

Comment:
is this used somewhere else in the classes? I would say take out random_roll? just to keep in a little simpler.

Reply:
OK

Reply:
Removed in 7d1cb86